sym-files2/ourdygraph.py

```diff
@@ -82,7 +82,8 @@ def _header(self):
             if self.ggs['dygraph_settings'].get('max_points') is not None:
                 max_points = int(self.ggs['dygraph_settings']['max_points'])
         # Calculate the data point reduction factor
-        self.reduction = (self.measurement_count / max_points) + 1
+        
+        self.reduction = int(self.measurement_count / max_points) + 1
```

Comment:
I think these could be handled by self.measurement_count // max_points but that is a minor difference